recipe/meta.yaml

```diff
@@ -33,6 +34,9 @@ requirements:
     - gshhg-gmt
     - dcw-gmt
   run:
+    - libblas
+    - libcblas
+    - liblapack
```

Comment:
These should have run_exports and should not be necessary to list them here. Does it fail without them?

Reply:
In #77, macOS build fails because liblapack is not installed/found, so I added liblapack to the list.
```
dyld: Library not loaded: @rpath/liblapack.3.dylib
  Referenced from: /usr/local/miniconda/conda-bld/gmt_1565715990180/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/libgmt.5.dylib
  Reason: image not found
```